db/knex_migrations/2024-08-24-0000-conditions.js

```diff
@@ -0,0 +1,12 @@
+exports.up = function (knex) {
+    return knex.schema
+        .alterTable("monitor", function (table) {
+            table.text("conditions").defaultTo(null);
```

Comment:
While we have a similar filed already (See https://github.com/louislam/uptime-kuma/pull/4717/files), this is likely a better implementation.
We can delete the other column when we refactor said Monitor or when jsonata support is added to the Operators..